contracts/test/river/token/base/RiverBase.t.sol

```diff
@@ -65,11 +64,19 @@ contract RiverBaseTest is BaseSetup, ILockBase, IOwnableBase {
     assertEq(riverFacet.allowance(alice, bob), amount);
   }
 
-  function test_permit(uint256 amount, address bob) external {
+  function test_fuzz_permit(
+    uint256 alicePrivateKey,
+    uint256 amount,
+    address bob
+  ) external {
+    alicePrivateKey = bound(
+      alicePrivateKey,
+      1,
+      0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364140
+    );
```

Comment:
https://ethereum.stackexchange.com/questions/164194/is-there-really-2256-private-key